### PR TITLE
Update tr.lua

### PR DESCRIPTION
**Pull request by NovaDiablox**
(2023-12-14)
  - Added all new translation strings

### CODE REVIEW

File: lua/terrortown/lang/tr.lua [hunk 1]
```diff
@@ -1868,118 +1869,118 @@ L.search_dmg_club = "Ceset çürümüş ve hırpalanmış. Belli ki dövülerek
...
+L.search_floor_flesh = "Kurbanın giysileri biraz nemli geliyor. Sanki ıslak bir yüzeye düşmüş gibi. Etli bir yüzey veya bir su kütlesinin kumlu zemini gibi."
+L.search_floor_grate = "Kurbanın derisi bifteğe benziyor. Belli bir sırada kalın çizgiler kurbanın her yerinde görülüyor. Izgaranın üzerinde mi dinlendiler?"
+L.search_floor_alienflesh = "Sence uzaylı eti mi? Kulağa biraz tuhaf geliyor ama dedektif yardımcısı kitabınız onu olası bir zemin yüzeyi olarak listeliyor."
+L.search_floor_snow = "İlk bakışta giysileri sadece ıslak ve buz gibi geliyor ama kenarlardaki beyaz köpüğü gördüğünüzde anlarsınız. Bu kar!"
+L.search_floor_plastic = "'Ah, bu acıtmış olmalı.' Cesetleri yanıklarla kaplıdır. Plastik bir yüzey üzerinde kayarken meydana gelenlere benziyorlar."
+L.search_floor_metal = "En azından artık öldükleri için tetanoz olamazlar. Yaralarını pas kaplıyor. Muhtemelen metal bir yüzeyde öldüler."
+L.search_floor_sand = "Küçük küçük pürüzlü taşlar soğuk cesetlerine yapışmış. Kumsaldaki kaba kum gibi. Ahh, her yere bulaşıyor!"
+L.search_floor_foliage = "Doğa harikadır. Kurbanın kanlı yaraları, neredeyse gizlenecek kadar yeşilliklerle kaplı.”
```
Comment (saibotk) [inline lua/terrortown/lang/tr.lua:1888]:
```suggestion
L.search_floor_foliage = "Doğa harikadır. Kurbanın kanlı yaraları, neredeyse gizlenecek kadar yeşilliklerle kaplı."
```

**Review by saibotk — CHANGES_REQUESTED** [review]

**Comment by saibotk** [comment]
Can you add a Changelog entry?

**Review by TimGoll — APPROVED** [review]

**Review by TimGoll — CHANGES_REQUESTED** [review]
Add the changelog to this PR, not a second PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
...
 ## Unreleased
 
+### Changed
+
+- Updated the Turkish localization file (by @NovaDiablox)
+- Added all new translation strings
+  
+### Fixed
+
+- Fixed context in some sentences
```
Comment (saibotk) [inline CHANGELOG.md:14]:
```suggestion
- Updated the Turkish localization file (by @NovaDiablox)
```

**Review by saibotk — CHANGES_REQUESTED** [review]
Otherwise this is confusing once more changes have been introduced

**Review by saibotk — APPROVED** [review]

**Review by TimGoll — APPROVED** [review]

**Pull request merged**